aws-logs-logstream/aws-logs-logstream.json

```diff
@@ -0,0 +1,43 @@
+{
+    "typeName" : "AWS::Logs::LogStream",
+    "description" : "Resource Type definition for AWS::Logs::LogStream",
+    "additionalProperties" : false,
+    "properties" : {
+        "LogGroupName" : {
+            "type" : "string",
+            "description" : "The name of the log group where the log stream is created."
+        },
+        "LogStreamName" : {
+            "type" : "string",
+            "description" : "The name of the log stream. The name must be unique wihtin the log group."
+        }
+    },
+    "required" : [ "LogGroupName", "LogStreamName" ],
+    "createOnlyProperties" : [ "/properties/LogGroupName", "/properties/LogStreamName"],
+    "primaryIdentifier" : [ "/properties/LogGroupName", "/properties/LogStreamName"],
+    "handlers" : {
+        "create" : {
+            "permissions" : [
+                "logs:CreateLogStream",
+                "logs:DescribeLogStreams"
+            ]
+        },
+        "read" : {
+            "permissions" : [
+                "logs:DescribeLogStreams"
+            ]
+        },
+        "delete" : {
+            "permissions" : [
+                "logs:DeleteLogStream",
+                "logs:DescribeLogStreams"
```

Comment:
`logs:DescribeLogStreams` Looks like an excessive permission for delete

Reply:
removed DescribeLogStreams permission